.github/workflows/deploy-sdk-armada-prod.yaml

```
@@ -0,0 +1,118 @@
+          DISCORD_EMBEDS:
+            '[{"title":"Link to GitHub Action:
+            ${{github.run_id}}","url":"https://github.com/${{github.repository}}/actions/runs/${{github.run_id
+            }}"}]'
+        with:
+          args:
+            'SDK deployment `${{github.run_id}}` started on armada prod 🚀 (from branch `${{
+            github.ref_name }}` by [${{github.triggering_actor
+            }}](https://github.com/${{github.triggering_actor }})).'
+
+      - name: Git clone the repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)